README.md

```diff
@@ -3,11 +3,36 @@
 [![Build Status](https://travis-ci.org/dewv/customerServiceApp.svg?branch=master)](https://travis-ci.org/dewv/customerServiceApp)
 [![Coverage Status](https://coveralls.io/repos/github/dewv/customerServiceApp/badge.svg?branch=master)](https://coveralls.io/github/dewv/customerServiceApp?branch=master)
 
-# Reference Data (Subject to updates)
-- Controller needs to accomplish:  
-  <ul><li>Handles HTTP requests, turning them into operations on model and/or view (staff or student, etc)</li>  
-  <li>RESTful API; defines "business operations" such as CRUD methods as HTTP requests (method, URL)</li></ul>  
-- View needs to create a useable webpage  
-  <ul><li>user interface that can handle CRUD operations</li></ul>  
-- Model needs to handle the data being utilized  
-  <ul><li>Uses the public API and database to store and read data from CRUD/HTTP requests</li></ul>
+# Design (Subject to updates)
+- All uses base view from submodule
+- Student/Staff submit new issue
+  - they get here through the nlc portal (makes most sense to me)
+  - anonymous was suggested, I suggest being logged in to help avoid random spam.
```

Comment:
We want this to be usable by *anyone*, even a random stranger who wants to submit an inquiry to us. (It can be a "contact" form as well as a "problem" form.) Spam is a concern, and we will probably do something about it eventually-- validate an email, use a CAPTCHA, whatever. For now, we can treat login as optional-- if you have D&E account, login, otherwise you are "anonymous".